### PR TITLE
✨ Frontend: S4L-Academic

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Assets and logic for the coming s4l-academic product

S4L-Academic

### CODE REVIEW

File: services/static-webserver/client/source/resource/osparc/s4lacad_splitimage.png
Comment (pcrespov) [inline services/static-webserver/client/source/resource/osparc/s4lacad_splitimage.png:1]: would it be possible to have something of a 10-20 KB instead?

**Review by pcrespov — APPROVED** [review]

File: services/static-webserver/client/source/resource/osparc/s4lacad_splitimage.png
Comment (odeimaiz) [reply services/static-webserver/client/source/resource/osparc/s4lacad_splitimage.png:1]: This is a placeholder, it should be replaced by the one provided by the design team

**Comment by SCA-ZMT** [comment]
is this the final product name? there's no .academic or .academia domain, only .academy

**Comment by odeimaiz** [comment]
> is this the final product name? there's no .academic or .academia domain, only .academy

This is just an internal product name. The one rendered in the frontend can be tuned in the DB.

**Review by elisabettai — APPROVED** [review]
Nice. Is that the final logo? It looks like the "A" in "ACADEMIC" is missing some transparency.

**Review by GitHK — APPROVED** [review]

**Comment by odeimaiz** [comment]
> Nice. Is that the final logo? It looks like the "A" in "ACADEMIC" is missing some transparency.

All placeholders. I try not to make them too pretty, so that they do get replaced 👌

**Review by YuryHrytsuk — APPROVED** [review]
:tada:

**Pull request merged**